STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -671,6 +671,90 @@ The initial contents of this registry are the extensions defined by the JSON Sch
 
 # Security Considerations
 
+## Trust Model
+
+The ACME trust model needs to be extended to include the bidirectional trust
+between NDC and IdO.  Note that once this trust link is established, it
+automatically becomes recursive.  Therefore, there has to be a trust
+relationship between each of the nodes in the delegation chain.  In case of
+cascading CDNs this is contractually defined; in other environments, the IdO
+might want to limit the parties that are indirectly delegated (see
```

Comment:
I would prefer stronger wording here: In all other environments, the IdO MUST have a well defined set of delegated identities, and enforce it as it forwards requests.

Reply:
Sure, and that is/should be encoded explicitly in the CSR template. But I don't think the MUST you are proposing would prevent an NDC colluded with a malicious third party to share the delegated identifier, which is what I was trying to say here.

Reply:
I agree, and enforcement by the NDC is "defense in depth".

Reply:
> if the IdO refuses to forward requests when it doesn't like the delegated identity, why is that not preventing this attack?

The attack is that IdO happily delegates its `foo.ido.example` name to NDC, and (rascal) NDC in turn delegates `foo.ido.example` to some other random guy which IdO is not happy to delegate the name.  The problem I want to highlight is that, given RFC6125 matching rules, it's not possible for IdO to exercise any further control over the usage of the name once the name is handed over to NDC.